llvm/test/tools/llvm-mca/RISCV/SpacemitX60/atomic.s

```diff
@@ -0,0 +1,312 @@
+# NOTE: Assertions have been autogenerated by utils/update_mca_test_checks.py
+# RUN: llvm-mca -mtriple=riscv64 -mattr=+rva22u64 -mcpu=spacemit-x60 -iterations=1 < %s | FileCheck %s
```

Comment:
I would recommend and prefer all the new scheduling model tests to use the more concise `-instruction-tables=full` instead of the default MCA view